specification/2.0/schema/material.schema.json

```diff
@@ -39,7 +39,30 @@
             "default": [ 0.0, 0.0, 0.0 ],
             "description": "The emissive color of the material.",
             "gltf_detailedDescription": "The RGB components of the emissive color of the material. If an emissiveTexture is specified, this value is multiplied with the texel values."
+        },
+        "alphaMode": {
+            "type": "string",
+            "enum": [ "OPAQUE", "MASK", "BLEND" ],
+            "default": "OPAQUE",
+            "description": "The alpha rendering mode of the material.",
+            "gltf_detailedDescription": "The material's alpha rendering mode enumeration specifying the interpretation of the alpha value of the main factor and texture. If the alpha mode is not specified, it defaults to `OPAQUE` where the alpha value is ignored and the rendered output is fully opaque. In `MASK` mode, the rendered output is either fully opaque or fully transparent depending on the alpha value and the specified alpha cutoff value. In `BLEND` mode, the alpha value is used to composite the source and destination areas. The rendered output is combined with the background using the normal painting operation (i.e. the Porter and Duff over operator)."
+        },
+        "alphaCutoff": {
+            "type": "number",
+            "minimum": 0.0,
+            "default": 0.5,
+            "description": "The alpha cutoff value of the material.",
+            "gltf_detailedDescription": "This value specifies the cutoff threshold when in `MASK` mode. If the alpha value is greater than or equal to this value then it is rendered as fully opaque, otherwise, it is rendered as fully transparent. This value is ignored for other modes."
+        },
+        "doubleSided": {
+            "type": "boolean",
+            "default": false,
+            "description": "Indicates whether a material is double sided.",
```

Comment:
nit: a material --> the material (to be consistent with the rest of the descriptions)